### PR TITLE
thief and rogue skills swapped, damage and AC factors updated

### PR DESCRIPTION
**Pull request by daedela**

### CODE REVIEW

File: src/act.informative.cpp [hunk 1]
```diff
@@ -3843,6 +3843,10 @@ ACMD(do_scan) {
     int from_room, to_room;
     Exit *exit;
 
+    if (GET_CLASS(ch) != CLASS_ROGUE) {
+        GET_HIDDENNESS(ch) = 0;
```
Comment (stridera) [inline src/act.informative.cpp:3847]: Why not just set this for all classes?  Are rogues the only ones that can't scan while hidden?

**Review by stridera — APPROVED** [review]

**Comment by ecourtt** [comment]
yes, this is something we're giving to rogues, that others can't do.

File: src/act.informative.cpp [hunk 2]
```diff
@@ -3843,6 +3843,10 @@ ACMD(do_scan) {
...
+    }
+
```
Comment (stridera) [inline src/act.informative.cpp:3849]: You should hide the "$N scans the area" room act below if they're hidden.

Comment (daedela) [reply src/act.informative.cpp:3849]: It is hidden.  I forgot to actually stop hiddenness from being removed

**Pull request merged**